src/js/color_change.js

```diff
@@ -145,6 +142,21 @@ $(function() {
             courseId = lastAddedCourse[0] + 1;
         }
 
+        var courseSplit = course.split('-');
+        var courseCode = courseSplit[0].trim();
+        var courseTitle = '';
+        var i;
+
+        for (i = 1; i < courseSplit.length - 1; ++i) {
+            courseTitle = courseTitle + courseSplit[i] + '-';
+        }
+
+        if (courseSplit.length > 1) {
+            courseTitle = courseTitle + courseSplit[i];
+        }
+
+        courseTitle = courseTitle.trim();

```

Comment:
I think this should work, plus lesser code and easier to understand. Can you test and confirm? If it works, please make the this change.

```suggestion
        var courseTitle = courseSplit.slice(1).join('');
```

Reply:
Yeah, this is better. I'll replace `join('')` with `join('-')`.